src/components/Input/Input.js

```diff
@@ -0,0 +1,150 @@
+import { useRef } from 'react';
+
+const InputText = props => {

```

Comment:
Please, use destructuring:
const { label, value, type, error, showError } = props;

and then you can use these variables